third_party/amd/lib/TritonAMDGPUToLLVM/DecomposeUnsupportedConversions.cpp

```diff
@@ -41,10 +41,8 @@ static int getCvtOpLDSUsage(triton::gpu::ConvertLayoutOp &cvtOp) {
   return bytes;
 }
 
-bool isPowerOfTwo(unsigned x) { return x && (x & (x - 1)) == 0; }
-
 static std::vector<std::pair<int, int>> factorizePowerOf2(int n) {
-  assert(isPowerOfTwo(n));
+  assert(llvm::isPowerOf2_32(n) && "n must be a power of 2");
```

Comment:
nit, I don't think "n must be a power of 2" adds anything to what's already in the assertion.